data/moves.js

```diff
@@ -3195,17 +3196,21 @@ exports.BattleMovedex = {
 			if (!target.volatiles['substitute'] || move.infiltrates) this.boost({evasion: -1});
 			let removeTarget = ['reflect', 'lightscreen', 'auroraveil', 'safeguard', 'mist', 'spikes', 'toxicspikes', 'stealthrock', 'stickyweb'];
 			let removeAll = ['spikes', 'toxicspikes', 'stealthrock', 'stickyweb'];
+			let success = false;
```

Comment:
This makes Defog fail if there are no screens or hazards, even if the evasion drop succeeds. Is that intentional?

Reply:
Nope. I think I may have misread that first line.